Source/m_menu.c

```diff
@@ -6272,6 +6272,12 @@ void M_ResetSetupMenu(void)
   {
     FLAG_SET_VANILLA(weap_settings1[i].m_flags, S_DISABLE);
   }
+
+  // [FG] exclusive fullscreen
+  if (fullscreen_width != 0 || fullscreen_height != 0)
+  {
+    gen_settings1[general_fullscreen].m_flags |= S_DISABLE;
```

Comment:
`gen_settings1` has a "Video" title menu item for which there is no value in the enum. So, you need to write `gen_settings1 [general_fullscreen + 1]` or change this enum:
https://github.com/fabiangreffrath/woof/blob/c9455a7d779faaa8c598d217c9582be8027c881d/Source/m_menu.c#L3464-L3478